packages/web-react/src/components/FileUploader/FileUploaderAttachment.tsx

```diff
@@ -49,6 +53,26 @@ const FileUploaderAttachment = (props: SpiritFileUploaderAttachmentProps) => {
 
   useFileUploaderAttachment({ attachmentRef, file, name, onError });
 
+  useDeprecationMessage({
+    method: 'property',
+    trigger: !!buttonLabel,
+    componentName: 'AttachmentDismissButton',
```

Comment:
As `AttachmentDismissButton` is internal component, I think this should be `FileUploaderAttachment`. And in the second deprecation too.